src/Umbraco.Infrastructure/BackgroundJobs/Jobs/WebhookFiring.cs

```diff
@@ -87,12 +87,11 @@ public async Task RunJobAsync()
         }));
     }
 
-    private async Task<HttpResponseMessage?> SendRequestAsync(Webhook webhook, string eventName, object? payload, int retryCount, CancellationToken cancellationToken)
+    private async Task<HttpResponseMessage?> SendRequestAsync(Webhook webhook, string eventName, string? serializedObject, int retryCount, CancellationToken cancellationToken)
     {
         using var httpClient = new HttpClient();
 
-        var serializedObject = _jsonSerializer.Serialize(payload);
-        var stringContent = new StringContent(serializedObject, Encoding.UTF8, "application/json");
+        var stringContent = new StringContent(serializedObject ?? string.Empty, Encoding.UTF8, "application/json");
```

Comment:
@Zeegaan we could use the `MediaTypeNames.Application.Json` constant here 👍

Reply:
Have to admit I did not know that was a thing, good point though 🙈

Reply:
.NET Core has some constants for some of the common mime types:
https://learn.microsoft.com/en-us/dotnet/api/system.net.mime.mediatypenames.application?view=net-7.0